.gitignore

```diff
@@ -0,0 +1,5 @@
+venv
+cinema/migrations
+db.sqlite3
+.idea
+app/settings.py
```

Comment:
It's unusual to ignore `settings.py` in version control. Consider using environment variables for sensitive data and including a `settings.py.example` file instead. This allows others to understand the necessary settings without exposing sensitive information.